Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -154,6 +154,14 @@ function setUpGeolocation() {
   polyfillGlobal('geolocation', require('Geolocation'), GLOBAL.navigator);
 }
 
+function setUpProduct() {
+  let rn_version = require('../../../package.json').version;
```

Comment:
nit: camel case rnVersion.

cc @vjeux will this package.json require work for facebook?